### ISSUE TITLE
Missing default http timeout

### ISSUE THREAD

**Issue by Nick233333** [xendit/xendit-php#67]
```php
    public function __construct(Guzzle $http = null)
    {
        if ($http) {
            $this->http = $http;
        } else {
            $baseUri = strval(Xendit::$apiBase);
            $this->http = new Guzzle(
                [
                    'base_uri' => $baseUri,
                    'verify' => false,
                    'timeout' => 60
                ]
            );
        }
    }
```

**Comment by Nick233333** [comment]
@stanleynguyen

**Comment by stanleynguyen** [comment]
Hey @Nick233333. Thanks for the suggestion. I think swapping out the whole httpClient would be a better solution.

**Comment by stanleynguyen** [comment]
Resolved with #69
